hello.html

```diff
@@ -0,0 +1,12 @@
+<!doctype html>
+<html lang="en">
+<head>
+<meta charset="UTF-8">
+<meta name="viewport" content="width=device-width, initial-scale=1">
+<title>Hello Microverse</title>
+<meta name="description" content="Our first page">
+<meta name="keywords" content="html tutorial template">
+<link rel="stylesheet" href="color.css">
+</head>
+<h1>"Hello Microverse!"</h1>
+</html>
```

Comment:
- [ OPTIONAL] Please, follow the Html best practice, your `</h1>` tag should be wrapped inside the body element, I would suggest you not type the HTML skeleton manually, generating it automatically will improve your productivity and it is still great of doing that to avoid errors like indentations or forgetting some `tags` as in your case, you can do it by using this command on your key-board `shift 1` then press enter or write `doc` inside of the HTML file and press enter, this should work too.